interceptor/tracing_interceptor.go

```diff
@@ -788,20 +818,71 @@ func (t *tracingWorkflowOutboundInterceptor) startNonReplaySpan(
 		},
 		ToHeader: true,
 		Time:     time.Now(),
-	})
+	}, t.root.workflowHeaderReader(ctx), headerWriter)
 	if err != nil {
-		return nopSpan{}, ctx, newErrFut(ctx, err)
+		return nopSpan{}, ctx, workflowFutureFromErr(ctx, err)
 	}
 	return span, newCtx, nil
 }
 
+type tracingNexusOperationInboundInterceptor struct {
+	NexusOperationInboundInterceptorBase
+	root *tracingInterceptor
+}
+
+// CancelOperation implements internal.NexusOperationInboundInterceptor.
+func (t *tracingNexusOperationInboundInterceptor) CancelOperation(ctx context.Context, input NexusCancelOperationInput) error {
+	info := nexus.ExtractHandlerInfo(ctx)
+	// Start span reading from header
+	span, ctx, err := t.root.startSpanFromContext(ctx, &TracerStartSpanOptions{
+		Operation:  "RunCancelNexusOperationHandler",
+		Name:       info.Service + "/" + info.Operation,
+		DependedOn: true,
+		FromHeader: true,
+		Time:       time.Now(),
+	}, t.root.nexusHeaderReader(input.Options.Header), t.root.headerWriter(ctx))
+	if err != nil {
+		return err
+	}
+	var finishOpts TracerFinishSpanOptions
+	defer span.Finish(&finishOpts)
+
+	err = t.Next.CancelOperation(ctx, input)
+	finishOpts.Error = err
+	return err
+}
+
+// StartOperation implements internal.NexusOperationInboundInterceptor.
+func (t *tracingNexusOperationInboundInterceptor) StartOperation(ctx context.Context, input NexusStartOperationInput) (nexus.HandlerStartOperationResult[any], error) {
+	info := nexus.ExtractHandlerInfo(ctx)
+	// Start span reading from header
+	span, ctx, err := t.root.startSpanFromContext(ctx, &TracerStartSpanOptions{
+		Operation:  "RunStartNexusOperationHandler",
```

Comment:
There are all sorts of other name options we can bikeshed over here (`RunCancelNexusOperation`, `HandleNexusCancelOperation`, etc), but so long as consistent across SDKs, I don't have a strong preference. I do think `Start<whatever>` for outbound and `Run<whatever>` or `Handle<whatever>` for inbound may make more sense, but doesn't matter much.